atd-vzv/src/Components/Popover/popoverConfig.js

```diff
@@ -2,8 +2,13 @@ import React from "react";
 import raceEthnicityTable2018 from "./raceEthnicityTable2018.png";
 import { popEsts } from "../../constants/popEsts";
 
-const formatNumber = (num) => {
-  return num.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,");
+/**
+ * Format a number with commas
+ * @param {Number} num - a number that needs commas added
+ * @returns {String} - a string of the number with commas added
+ */
+const addCommasToNumber = (num) => {
+  return num ? num.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,") : "";
```

Comment:
Not sure what fallback value we want here, but the deploy preview broke when `undefined` was coming through function so I wanted to do something about it.

Reply:
just want to point out that we handle this w/o regex in Moped using [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat).

```javascript
> new Intl.NumberFormat('en-US').format(123567890)
// '123,567,890'
```

Reply:
...and actually if you don't specify the locale, i think the number format is [localized automatically](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locale_identification_and_negotiation) - so we might actually want to opt for this in case someone is browsing our website....from India?

Reply:
btw @patrickm02L if this is scheduled to fail again over the weekend (which i think you mentioned was likely) — i am all for merging as-is!